docs/ProgrammingConcerns.md

```diff
@@ -0,0 +1,170 @@
+# Please read this before contributing to FEX-Emu
```

Comment:
This is the wrong place for that kind of note. The section header and the paragraph don't really add anything, and instead a CONTRIBUTING.md could be created to point developers to important documents like this one.

Reply:
Removed this.